src/asf_tools/__main__.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+import sys
+from importlib.metadata import entry_points
+
+
+def main():
+    parser = argparse.ArgumentParser(prefix_chars='+', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '++process', choices=['water_map', 'flood_map'], default='water_map',
+        help='Select the HyP3 entrypoint to use'  # HyP3 entrypoints are specified in `pyproject.toml`
+    )
+
+    args, unknowns = parser.parse_known_args()
+    # NOTE: Cast to set because of: https://github.com/pypa/setuptools/issues/3649
```

Comment:
I'm confused by the discussion on that issue. It sounds like maybe it was fixed in newer versions of `setuptools`? Is the issue that we don't necessarily have control over which version of `setuptools` the user has installed locally?

Reply:
I kept getting duplicated entry points with an editable install. I believe there's (3) things going on in that discussion:
1. importlib_metadata, the backport for python <3.9(?), reports duplicate entry points, and was fixed
2. importlib.metadata, in the standard library, still reports duplicate entypoints
3. it's really a weird decision pip/setuptools took with editable installs, and so stdlib authors think it's their problem to fix and likely won't get resolved anytime soon because of it.